.github/workflows/main.yml

```diff
@@ -80,31 +70,25 @@ jobs:
       id-token: write
 
     steps:
-      - uses: actions/checkout@v3
-        with:
-          token: ${{ secrets.GH_TOKEN }}
+      - uses: actions/checkout@v4
 
-      - name: Set up Python
-        uses: actions/setup-python@v4
-        with:
-          python-version: "3.9"
+      - name: Install uv
+        uses: astral-sh/setup-uv@v3
 
       - name: Check release
         id: check_release
         run: |
-          python -m pip install --upgrade pip
-          python -m pip install autopub[github]
-          autopub check
+          uvx --with 'autopub[github]' autopub check
 
       - name: Publish
         if: ${{ steps.check_release.outputs.autopub_release=='true' }}
         env:
-          GITHUB_TOKEN: ${{ secrets.GH_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
-          autopub prepare
-          autopub commit
-          autopub build
-          autopub githubrelease
+          uvx --with 'autopub[github]' autopub prepare
+          uvx --with 'autopub[github]' autopub commit
+          uvx --with 'autopub[github]' autopub build
+          uvx --with 'autopub[github]' autopub githubrelease
```

Comment:
I don't think this latter step will succeed without something akin to https://github.com/getpelican/cookiecutter-pelican-plugin/blob/f4876b6192b13d6d05f7d89e17e219957efe15bd/%7B%7Bcookiecutter.repo_name%7D%7D/.github/workflows/main.yml#L82, and I don't know how to do that with `uvx --with […]` because I don't have much experience with `uv` yet.

Reply:
Since https://github.com/autopub/autopub/pull/45 has been merged, and AutoPub 0.4.0 released, the aforementioned comment should hopefully no longer apply 🤞